exp/operator/api/v1alpha4/bootstrapprovider_types.go

```diff
@@ -33,6 +33,7 @@ type BootstrapProviderStatus struct {
 // +kubebuilder:object:root=true
 
 // BootstrapProvider is the Schema for the bootstrapproviders API
+// nolint: maligned
```

Comment:
We just merged a PR in main removing thin linter, now deprecated (see https://github.com/kubernetes-sigs/cluster-api/pull/4353#discussion_r599067826).
Might be better to rebase instead of adding //nolints.

Reply:
thanks!

Reply:
I can include the rebase here, I don't think we can merge anything in the operator branch otherwise.